exporter/opentelemetry-exporter-otlp-proto-common/src/opentelemetry/exporter/otlp/proto/common/_internal/trace_encoder/__init__.py

```diff
@@ -142,7 +142,7 @@ def _encode_events(
                 name=event.name,
                 time_unix_nano=event.timestamp,
                 attributes=_encode_attributes(event.attributes),
-                dropped_attributes_count=event.attributes.dropped,
+                dropped_attributes_count=event.dropped_attributes,
```

Comment:
I think there are some tests [here](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/tests/trace/test_trace.py) that use `attribute.dropped`. Although your change won't break this, what do you think about updating to use this new property as well?

Reply:
Yes, noticed it while adding the unit test. Changes made in the new commit.